code/modules/mob/living/simple_animal/bees/bees_datums.dm

```diff
@@ -54,6 +57,7 @@
 	qdel(src)
 
 /datum/bee/Destroy()
+	bees_count--
```

Comment:
This should be in `Del()`, not `Destroy()`
It shouldn't really matter but is more foolproof that way